ganglia.php

```diff
@@ -350,7 +350,7 @@ function Gmetad () {
       }
 
    if ($debug) print "<br/>DEBUG: Creating parser\n";
-   if ( in_array($context, $SKIP_GMETAD_CONTEXTS) ) {
+   if ( isset($context) && is_array($context) && isset($SKIP_GMETAD_CONTEXTS) && is_array($SKIP_GMETAD_CONTEXTS) && in_array($context, $SKIP_GMETAD_CONTEXTS) ) {
```

Comment:
fix:  PHP Fatal error:  Uncaught TypeError: in_array(): Argument #2 ($haystack) must be of type array, null given in /usr/share/ganglia/ganglia.php:353